e2e-tests/ukef/cypress/e2e/journeys/portal/deal-cancellation/gef/deal-cancellation-maker-unable-issue-facility.spec.js

```diff
@@ -0,0 +1,107 @@
+
+        cy.createGefFacilities(deal._id, [facilities[index]], BANK1_MAKER1).then((createdFacilities) => {
+          dealFacilities.push(createdFacilities.details);
+        });
+        allDealFacilities.push(dealFacilities);
+      });
+    });
+  });
+
+  beforeEach(() => {
+    cy.clearCookie('dtfs-session');
+    cy.clearCookie('_csrf');
+    cy.getCookies().should('be.empty');
+  });
+
+  after(() => {
+    cy.clearCookies();
+    cy.clearCookie('dtfs-session');
+    cy.clearCookie('_csrf');
+    cy.getCookies().should('be.empty');
+  });
+
+  describe('Deal cancellations on tfm with effective dates in the past and in the future', () => {
+    before(() => {
+      dealIdsWithEffectiveDate.forEach((dealIdWithEffectiveDate) => {
+        cy.gefDealCancellationFlow(dealIdWithEffectiveDate.id, dealIdWithEffectiveDate.effectiveDate);
+      });
+      cy.clearCookie('dtfs-session');
+      cy.clearCookie('_csrf');
+      cy.getCookies().should('be.empty');
+      cy.login(BANK1_MAKER1);
+    });
+
+    it('Gef AIN deal with issued facilities is submitted to UKEF, user cancel deal in the future in TFM. Maker unable to issue facility on portal', () => {
+      gefPages.applicationDetails.visit(dealIdsWithEffectiveDate[0].id);
+      gefPages.applicationPreview.unissuedFacilitiesHeader().should('not.exist');
+    });
+
+    it('Gef AIN deal with issued facilities is submitted to UKEF, user cancel deal in the past in TFM. Maker unable to issue facility on portal', () => {
+      gefPages.applicationDetails.visit(dealIdsWithEffectiveDate[1].id);
+      gefPages.applicationPreview.unissuedFacilitiesHeader().should('not.exist');
+    });
+
+    it('Gef AIN deal with unissued facilities is submitted to UKEF, user cancel deal in the future in TFM. Maker unable to issue facility on portal', () => {
+      gefPages.applicationDetails.visit(dealIdsWithEffectiveDate[2].id);
+      gefPages.applicationPreview.unissuedFacilitiesHeader().should('not.exist');
+    });
+
+    it('Gef AIN deal with unissued facilities is submitted to UKEF, user cancel deal in the past in TFM. Maker unable to issue facility on portal', () => {
+      gefPages.applicationDetails.visit(dealIdsWithEffectiveDate[3].id);
+      gefPages.applicationPreview.unissuedFacilitiesHeader().should('not.exist');
+    });
+
+    it('Gef MIN deal with issued facilities is submitted to UKEF, user cancel deal in TFM. Maker unable to issue facility on portal', () => {
+      gefPages.applicationDetails.visit(dealIdsWithEffectiveDate[4].id);
+      gefPages.applicationPreview.unissuedFacilitiesHeader().should('not.exist');
+    });
+
+    it('Gef MIN deal with issued facilities is submitted to UKEF, user cancel deal in the past in TFM. Maker unable to issue facility on portal', () => {
+      gefPages.applicationDetails.visit(dealIdsWithEffectiveDate[5].id);
+      gefPages.applicationPreview.unissuedFacilitiesHeader().should('not.exist');
+    });
```

Comment:
I think this header wont exist even if the deal hasnt been cancelled (since its for unissued faciltiies) - we can leave them in but i dont think theyre adding much value